tests/rt.conf

```diff
@@ -51,7 +51,7 @@ COMPILE | s2swa_32bit_pdlib_sfs  | intel | -DAPP=S2SWA -D32BIT=ON -DHYDRO=ON -DC
 RUN | cpld_control_sfs                                  | - noaacloud                          | baseline |
 
 COMPILE | s2swa_32bit_pdlib_debug | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON | - noaacloud jet | fv3 |
-RUN | cpld_debug_gfsv17                                 | - noaacloud jet                      | baseline |
+RUN | cpld_debug_gfsv17                                 | + hera gaeac6 wcoss2                 | baseline |
```

Comment:
@RatkoVasic-NOAA  - This is commented out, so I need to re-run my tests... or do this test as a second step.  Same for cpld_debug_pdlib_p8

Reply:
@JessicaMeixner-NOAA thanks for doing tests! I suggest you just take subset of tests in new xxx.conf, and run rt.sh with -l xxx.conf option.

Reply:
Doing that now...

Reply:
@JessicaMeixner-NOAA how long are `s2swa_32bit_pdlib_debug_intel` or `2swa_32bit_pdlib_debug_intel` taking to compile for you? it's ~23 mins on ursa. is this normal?

Reply:
Seems long based on what i just got on hercules.  I think the log files have compile time output to them -- but not 100% sure what the numbers mean, so you could look there for more information.